hugo/package.json

```diff
@@ -25,17 +24,29 @@
     "@types/mustache": "^4.2.1",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.9",
-    "langium": "0.5.0-next.52ab085",
-    "langium-statemachine-dsl": "0.5.0-next.f2b3802",
-    "lz-string": "^1.4.4",
-    "monaco-editor-workers": "0.34.2",
-    "monaco-editor-wrapper": "1.3.2",
-    "vite": "^3.2.4",
-    "vscode-languageserver": "8.0.2"
+    "esbuild": "^0.15.7"
   },
   "dependencies": {
     "clsx": "^1.2.1",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "@babel/cli": "~7.19.3",
+    "@babel/core": "~7.20.5",
+    "@babel/preset-react": "~7.18.6",
+    "@babel/preset-typescript": "~7.18.6",
+    "@typefox/monaco-editor-react": "1.0.0-next.6",
+    "hugo-extended": "~0.89.4",
+    "langium": "1.1.0",
+    "langium-arithmetics-dsl": "~1.0.0",
+    "langium-domainmodel-dsl": "~1.0.0",
+    "langium-statemachine-dsl": "~1.0.0",
```

Comment:
The showcase doesn't run. I assume this is the culprit? These versions should all be the same.

Reply:
Only statemachine is implemented yet. Does statemachine work for you?

Reply:
The showcase looks like this:

![image](https://user-images.githubusercontent.com/4377073/224825287-91fa6ca8-66ed-4725-bd9b-012e2ce5da31.png)

Note that between 1.0.0 and 1.1.0 of Langium, the framework contains breaking changes which likely lead to runtime errors. If you're using langium `1.1.0`, you should use the same version for `langium-statemachine-dsl`.

Reply:
I'm pretty sure that's a tailwinds thing, I'm certain I saw this before as well, and thought it was fixed in here already 🤔 . I believe Monaco & everything is there, but the styling makes it effectively invisible, otherwise the states would never have populated in the first place.

Reply:
Fixed the Tailwind config file, just a one-liner.